### ISSUE TITLE
Install sqlite3 instead sqlite

### ISSUE THREAD

**Issue by johnking** [sequelize/sequelize-auto#221]
If we follow README.md, I found I need use `sqlite3` instead of `sqlite`, otherwise, I ended with an error: 

```
     throw new Error('Please install sqlite3 package manually');
```

**Comment by boxsnake** [comment]
@xeoshow `sqlite3` should be installed globally to make global `sequelize-auto` work.
```bash
npm i -g sqlite3
```

**Comment by steveschmitt** [comment]
Fixed in `sequelize-auto@0.5.1`.
